samples/visual-indicator/nodejs/pages/hello.html

```diff
@@ -0,0 +1,4 @@
+
+<div class="font-semibold font-title" style="color:gray">Hello, Welcome to Microsoft Teams</div>
```

Comment:
Update this "Hello, Welcome to Microsoft Teams"

Reply:
fixed as per comment